src/swarmus_example_pkg/CMakeLists.txt

```diff
@@ -0,0 +1,219 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(swarmus_example_pkg)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```

Comment:
Do we need to leave all the comments here?

Reply:
There are still many options that I don't really master for now, so I left the comments intentionally. I would be more comfortable if we kept the comments, since they also provide example commands and syntax.

But in the long run, yes, this would definitively be removed.

Reply:
I went through the file and removed a lot of stuff.